### PR TITLE
Refactor/auto compile

### PR DESCRIPTION
**Pull request by chylli-deriv**
Compile perl with CI system

### CODE REVIEW

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

File: .github/workflows/build_workflow.yml
Comment (qiangliu-deriv) [inline .github/workflows/build_workflow.yml:1]: I think rebuild deriv/perl , debian-pg, based on the new perl is not incluced in this task

Comment (chylli-deriv) [reply .github/workflows/build_workflow.yml:1]: I removed them from card description. we can do that in other cards.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
 # perl
-Custom compiled perl for Binary.com
 
-# compiling steps
+Custom compiled perl for Binary.com
```
Comment (chylli-deriv) [inline README.md]:
```suggestion
Custom compiled perl for deriv.com
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+## compiling steps
 
-We defined `-Dusesitecustomize` and removed `-Dusethreads`. Most of other parameters are copied from `perl -V | grep config_args`
-Run following command:
-```
-./Configure -Dusesitecustomize -Dinc_version_list=none -Dprefix=/home/git/binary-com/perl -Dvendorprefix=/home/git/regentmarkets/cpan/local -Dvendorlib=/home/git/regentmarkets/cpan/local/lib/perl5 -Dvendorarch=/home/git/regentmarkets/cpan/local/lib/perl5/x86_64-linux -Duselargefiles -Dccflags="-DDEBIAN -D_FORTIFY_SOURCE=2 -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security" -Dldflags=" -Wl,-z,relro" -Dlddlflags="-shared -Wl,-z,relro" -Dcccdlflags="-fPIC" -Duse64bitint -Dman1ext=1 -Dman3ext=3perl -Dpager=/usr/bin/sensible-pager -Uafs -Ud_csh -Ud_ualarm -Uusesfio -Uusenm -Uuseithreads -Uusemultiplicity -Ui_libutil -DDEBUGGING=-g -Doptimize=-O2 -Duseshrplib -des
-make
-make install
+```bash
+VERSION=5.26.1 ./rebuild.sh
```
Comment (mukesh-deriv) [inline README.md]: should we reference 5.26 here?

Comment (chylli-deriv) [inline README.md]:
```suggestion
VERSION=5.26.2 ./rebuild.sh
```

File: .github/workflows/build_workflow.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+name: Build workflow
+run-name: Build workflow
+on:
+  push:
```
Comment (lim-deriv) [inline .github/workflows/build_workflow.yml:4]:
```suggestion
  push:
    branches:
      - '!master'
```

**Pull request merged**